vuepress/docs/next/docs/reference/backup-restore.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebarDepth: 2
+---
+target=restored
+original=source
+# Backup and Restore an Entando Application
+
+This guide provides general instructions on how to backup and restore an existing Entando Application. The process entails the migration of databases, filesystem data, Keycloak instance, and bundles, plus the reconfiguration of the service address. 
+
+### Initial Considerations to Note
+* The restored instance can be on another cluster or a different namespace of the same cluster hosting the original Entando instance. 
+* The source and restored instances cannot be executed simultaneously, and data cannot be shared between them.
+* Bundles are externally deployed to an Entando Application and therefore must be reinstalled after the new instance is built. More details are below.
+
+## Copy and Migrate Databases
+
+1. Make a copy of the source databases or schemas using the native DBMS tools. Some standard Entando database schemas to back up are `prod_dedb`, `prod_kc_db`, `prod_portdb`, and `prod_servdb`, in addition to any that were created for your microservices. Those will follow a similar naming convention, with the microservice name followed by the suffix `_plugindb`.
+      * If the original database setup used a **custom resource**, refer to the [Database Custom Resource](https://developer.entando.com/v7.1/docs/reference/database-cr.html) document. 
+      * If it was provisioned using **environment variables** (PORTDB_*, SERVDB_* variables) in the EntandoApp custom resource, see the [External DB](https://developer.entando.com/v7.1/tutorials/devops/external-db.html) document for more information.  
+ 
+2. Restore the backups to the final DBMS.
+3. Install the restored instance by copying the data as needed.
+
+## Backup and Extract the AppEngine Filesystem Data
+Use tar commands to archive and extract the files, with kubectl providing support.
+1. Use a temporary directory to store the tarball of the data files for the AppEngine pod `quickstart-deployment`:
+
+```
+cd /tmp
+tar -czvp -f entando-data.tar.gz /entando-data/protected /entando-data/resources
+```
+
+2. Copy the generated tarball file from the original instance:
+
+```
+kubectl cp quickstart-deployment:/tmp/entando-data.tar.gz entando-data.tar.gz
+```
+
+3. Push it to the restored instance:
+
+```
+kubectl cp entando-data.tar.gz quickstart-deployment:/tmp/entando-data.tar.gz
```

Comment:
This is odd because it has the user copy the tar back into the identical pod. I assume it should be copied into the target/restored pod instead.